lib/module.js

```diff
@@ -10,14 +10,19 @@ const {
   flushCompileCache,
   getCompileCacheDir,
 } = require('internal/modules/helpers');
+const {
+  findPackageJSON,
+} = require('internal/modules/package_json_reader');
 const { stripTypeScriptTypes } = require('internal/modules/typescript');
 
 Module.findSourceMap = findSourceMap;
 Module.register = register;
```

Comment:
If we move `stripTypeScriptTypes` (to be in alphabetical order I presume), we might as well move those two

Reply:
I agree, but this is close to landing, so can this be handled in an immediate follow-up?